frontend/www/js/omegaup/components/user/SeeProfile.vue

```diff
@@ -195,7 +196,7 @@ import { Problem, ContestResult } from '../../linkable_resource';
     'omegaup-countryflag': country_Flag,
   },
 })
-export default class UserProfile extends Vue {
+export default class UserSeeProfile extends Vue {
```

Comment:
```suggestion
export default class ViewProfile extends Vue {
```

de preferencia la clase se debe llamar como el archivo.

Reply:
Listo